CHANGELOG.asciidoc

```diff
@@ -3,6 +3,112 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.6.0]]
+=== Beats version 7.6.0
+https://github.com/elastic/beats/compare/v7.5.1...v7.6.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Allow Metricbeat's beat module to read monitoring information over a named pipe or unix domain socket. {pull}14558[14558]
+- Remove version information from default ILM policy for improved upgrade experience on custom policies. {pull}14745[14745]
+- Running `setup` cmd respects `setup.ilm.overwrite` setting for improved support of custom policies. {pull}14741[14741]
+- Libbeat: Do not overwrite agent.*, ecs.version, and host.name. {pull}14407[14407]
+- Libbeat: Cleanup the x-pack licenser code to use the new license endpoint and the new format. {pull}15091[15091]
+- Users can now specify `monitoring.cloud.*` to override `monitoring.elasticsearch.*` settings. {issue}14399[14399] {pull}15254[15254]
+- Refactor metadata generator to support adding metadata across resources {pull}14875[14875]
+- Update to ECS 1.4.0. {pull}14844[14844]
+
+*Filebeat*
+
+- CEF extensions are now mapped to the data types defined in the CEF guide. {pull}14342[14342]
+
+*Journalbeat*
+
+- Remove broken dashboard. {pull}15288[15288]
+
+*Metricbeat*
+
+- Update cloudwatch metricset mapping for both metrics and dimensions. {pull}15245[15245]
```

Comment:
@kaiyan-sheng Is this really a breaking change?

Reply:
@urso Yes, I believe so. cloudwatch metricset is in GA and we changed the mapping including metric names and dimension names.